blog/2024-05-16-blog-status-of-lbse-in-webkit.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Status of LBSE in WebKit"
+author: rbuis
+permalink: /blog/status-of-lbse-in-webkit.html
+thumbnail: /assets/lbse-logo-wide.png
+preview: A look back at the last 7 months of development on LBSE in WebKit.
+---
+
+This blog entry gives an update on upstreaming and development of LBSE in WebKit in the last 7 months. For an explanation of
```

Comment:
Same here - first occurrence should probably say "Layer Based SVG Engine (LBSE)"  to introduce the initialism